Sprint-2/implement/contains.test.js

```diff
@@ -7,9 +7,16 @@ particular property
 E.g. contains({a: 1, b: 2}, 'a') // returns true
 as the object contains a key of 'a'
 
+
 E.g. contains({a: 1, b: 2}, 'c') // returns false
 as the object doesn't contains a key of 'c'
 */
+test("object contains the property", () => {
+  expect(contains({ a: 1, b: 2 }, "a")).toBe(true);
```

Comment:
Try checking also 
`expect(contains({ a: 1, b: 2 }, "b")).toBe(true);`
to discover bug in your code.

Reply:
thanks for this comment, I found the bug and amended the function